src/Sylius/Behat/Context/Setup/CliContext.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Bundle\InstallerBundle\Command\InstallSampleDataCommand;
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Helper\DialogHelper;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class CliContext implements Context
+{
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var DialogHelper
+     */
+    private $dialog;
+
+    /**
+     * @var SetupCommand
+     */
+    private $command;
+
+    /**
+     * @var array
+     */
+    private $inputChoices = [
+                'currency' => '',
+                'name' => ' Name',
+                'surname' => ' Surname',
+                'e-mail' => ' test@email.com',
+                'password' => ' pswd',
+                'confirmation' => ' pswd',
+             ];
+
+    /**
+     * @param KernelInterface $kernel
+     */
+    public function __construct(KernelInterface $kernel)
+    {
+        $this->kernel = $kernel;
+    }
+
+    /**
+     * @When I run Sylius CLI installer
+     */
+    public function iRunSyliusCommandLineInstaller()
+    {
+        $this->application = new Application($this->kernel);
+        $this->application->add(new SetupCommand());
+
+        $this->command = $this->application->find('sylius:install:setup');
+        $this->tester = new CommandTester($this->command);
+
+        $this->iExecuteCommandWithInputChoices('sylius:install:setup');
+    }
+
+    /**
+     * @Given I run Sylius Install Load Sample Data command
+     */
+    public function iRunSyliusInstallSampleDataCommand()
+    {
+        $this->application = new Application($this->kernel);
+        $this->application->add(new InstallSampleDataCommand());
+        $this->command = $this->application->find('sylius:install:sample-data');
+        $this->tester = new CommandTester($this->command);
+    }
+
+    /**
+     * @Given I confirm loading sample data
+     */
+    public function iConfirmLoadingData()
+    {
+        $this->iExecuteCommandAndConfirm('sylius:install:sample-data');
+    }
+
+    /**
+     * @Then /^I should see output "(.+)"$/
```

Comment:
`/^I should see output "([^"]+)"$/` or `I should see output :text`

The current one will match `I should see output "foo" and "bar"` step and pass `foo" and "bar` as `$text`.